src/platform/forms-system/src/js/validation.js

```diff
@@ -275,7 +280,7 @@ export function isValidForm(form, pageList, isTesting = false) {
         if (arrayData) {
           const itemsToKeep = arrayData.map(itemFilter || (() => true));
           // Remove the excluded array data
-          formData = _.set(
+          formData = set(
             arrayPath,
             arrayData.filter((item, index) => itemsToKeep[index]),
             formData,
```

Comment:
Looking above, it looks like it's safe to do `formData[arrayPath]`. Since we're assigning the return value to `formData` anyway we can skip a step:

```js
          formData[arrayPath] = arrayData.filter((item, index) => itemsToKeep[index]);
```